src/functions-and-arrays.js

```diff
@@ -1,41 +1,118 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
-
+function maxOfTwoNumbers(a, b) {
+  if (a>b){
+    return a
+  }
+  else{
+    return b
+  }
+}
+console.log(maxOfTwoNumbers(10,20))
 
 
 // Iteration #2: Find longest word
-const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
-function findLongestWord() {}
+const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
+function findLongestWord(str) {
+  let longestWord = " ";
+  for (let i = 0; i < words.length; i++){
+    if (words[i].length > longestWord.length){
+      longestWord = words[i];
+    }
+  }
+  return longestWord;
+}
 
+let result = findLongestWord(words)
```

Comment:
You don't need to call your functions, because they will be called by the jasmin tests. If you want to test your code, that's fine. But normally you should remove these function calls before you submit your solution.